Tests/OptimizelyTests-APIs/OptimizelyClientTests_Decide.swift

```diff
@@ -0,0 +1,76 @@
+/****************************************************************************
+* Copyright 2020, Optimizely, Inc. and contributors                        *
+*                                                                          *
+* Licensed under the Apache License, Version 2.0 (the "License");          *
+* you may not use this file except in compliance with the License.         *
+* You may obtain a copy of the License at                                  *
+*                                                                          *
+*    http://www.apache.org/licenses/LICENSE-2.0                            *
+*                                                                          *
+* Unless required by applicable law or agreed to in writing, software      *
+* distributed under the License is distributed on an "AS IS" BASIS,        *
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+* See the License for the specific language governing permissions and      *
+* limitations under the License.                                           *
+***************************************************************************/
+    
+import XCTest
+
+class OptimizelyClientTests_Decide: XCTestCase {
+
+    var optimizely: OptimizelyClient!
+
+    override func setUp() {
+        super.setUp()
+        
+        let datafile = OTUtils.loadJSONDatafile("api_datafile")!
+        optimizely = OptimizelyClient(sdkKey: OTUtils.randomSdkKey)
+        try! optimizely.start(datafile: datafile)
+    }
+   
+    func testCreateUserContext() {
```

Comment:
Please add one more test, in which after creating user context, we should modify attributes and see those changes should not be reflected. 

e.g.
attributes["country"] = "uk"
and check user.attributes[country] shouldn't reflect that change.